app/components/external-link.js

```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  tagName: "a",
+  href: "#",
+
+  click: function() {
+    window.open(this.decodeLink(), "_system");
```

Comment:
@swatijadhav Looking at the [README](https://github.com/apache/cordova-plugin-inappbrowser) for cordova-plugin-inappbrowser it appears window.open won't be around for much longer. Should we be using "cordova.InAppBrowser.open" instead?